src/components/Select/Select.js

```diff
@@ -146,33 +141,37 @@ class Select extends React.PureComponent {
       className,
       popperProps,
       name,
-      keepProvidedValue,
       ...rest
     } = this.props;
     const combinedLoading = loading || isLoading;
-    const combinedPlaceholder = noneText !== undefined ? noneText : placeholder;
+    let combinedPlaceholder = noneText !== undefined ? noneText : placeholder;
+    let justOpened = false;
```

Comment:
Can you add comments explaining this behavior? I don't understand it, and I'm a little worried about adding complexity with side effects in render.